setup.py

```diff
@@ -132,7 +131,7 @@
     keywords='copulas',
     name='copulas',
     packages=find_packages(include=['copulas', 'copulas.*']),
-    python_requires='>=3.7,<3.12',
+    python_requires='>=3.8,<3.12',
```

Comment:
Should we also update this line:
https://github.com/sdv-dev/Copulas/blob/a863cdd578d6d07d667a4a1d1bf44f69f5aa9ec8/setup.py#L15